WORKSPACE.bzlmod

```diff
@@ -15,9 +15,13 @@ local_repository(
 #)
 
 # Needed for making our release notes
-#load("//toolchains/git:git_configure.bzl", "experimental_find_system_git")
-#
-#experimental_find_system_git(
-#    name = "rules_pkg_git",
-#    verbose = False,
-#)
+load("//toolchains/git:git_configure.bzl", "experimental_find_system_git_bzlmod")
+
+experimental_find_system_git_bzlmod(
```

Comment:
Is it possible to stop depending on git instead?

Reply:
I'm trying build tooling that helps create release notes. I have not abandoned that idea yet. I may eventually, or move it to another project.  OTOH, keeping it here as an optional module might be the right thing because people so often want to get git-hash into the names of their release artifacts.

Reply:
Do we really have to do this through a toolchain? I would also prefer a simpler solution if possible.